doc/specs/stdlib_kinds.md

```diff
@@ -16,17 +16,24 @@ The `stdlib_kinds` module provides kind parameters for the Fortran intrinsic dat
 
 ### `sp`
 
-Alias for intrinsic named constant `real32` imported from `iso_fortran_env`.
+Provides real kind parameter for floating point numbers with a minimal precision of 6 significant digits.
```

Comment:
Should we document what the constants `sp` stands for here? Such as

`sp` stands for "single precision". Is `xdp` "extended double precision"?